lib/wip/product_mapping_exporter.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'sequel'
+
+module Magento
+  class ProductMappingExporter
```

Comment:
Wondering if we should turn this into a thor subcommand.

Reply:
Would you ever want to run this without also running `export`? I'm thinking maybe this can be called from other code that is part of a Thor subcommand, and the user never has to run this themselves. But maybe (for example if it is long-running) it is better for the user to be able to run them separately.

Reply:
I agree that this doesn't make sense to run in isolation, It makes sense that there'd be a single public subcommand that is called that is calling this exporter somewhere internally.